charts/datadog/README.md

```diff
@@ -1033,10 +1033,12 @@ helm install <RELEASE_NAME> \
 | otelAgentGateway.volumeMounts | list | `[]` | Specify additional volumes to mount in the otel-agent container |
 | otelAgentGateway.volumes | list | `[]` | Specify additional volumes to mount in the otel-agent container |
 | providers.aks.enabled | bool | `false` | Activate all specificities related to AKS configuration. Required as currently we cannot auto-detect AKS. |
+| providers.eks.controlPlaneMonitoring | bool | `false` | Enable control plane monitoring checks in the EKS cluster. |
 | providers.eks.ec2.useHostnameFromFile | bool | `false` | Use hostname from EC2 filesystem instead of fetching from metadata endpoint. |
 | providers.gke.autopilot | bool | `false` | Enables Datadog Agent deployment on GKE Autopilot |
 | providers.gke.cos | bool | `false` | Enables Datadog Agent deployment on GKE with Container-Optimized OS (COS) |
 | providers.gke.gdc | bool | `false` | Enables Datadog Agent deployment on GKE on Google Distributed Cloud (GDC) |
+| providers.openshift.controlPlaneMonitoring | bool | `false` | Enable control plane monitoring checks in the OpenShift cluster. |
```

Comment:
Add a note similar to `NOTES.txt` that configuring the etcd check reequires additional certificates